### DISCUSSION TITLE
Can't get the panzoom example to work

### DISCUSSION THREAD

**Discussion by Glyn-Caon** [fancyapps/ui#228] [Q&A]
I'm unable to get panzoom to work outside of the fancyapps.com playground.

Here is the panzoom quickstart example running correctly in the fancyapps playround: [https://fancyapps.com/playground/1Zs](https://fancyapps.com/playground/1Zs)

Here's an identical example not working via jsfiddle: [https://jsfiddle.net/woajdhuL/1/](https://jsfiddle.net/woajdhuL/1/)
console.log(Panzoom) returns a sensible object.

The JSfiddle is representitive of what I'm seeing in my project, what am I doing wrong?

**Comment by fancyapps** [comment]
Hi,

Sorry about the confusion, I will update docs to make it clear that you have to include panzoom.css file to make it work.

Demo - https://jsfiddle.net/psbkj3zq/
